test/Build.skopeo-umoci

```diff
@@ -13,6 +13,7 @@
 # issue #332 are still tested.
 #
 # ch-test-scope: standard
+# ch-test-builder-required: none
```

Comment:
So this one only works with `--builder=none`? Shouldn't it work with all builders, because no builder is used?